sgp4/tests.py

```diff
@@ -85,6 +86,44 @@ def test_satrec_attributes(self):
         self.assertEqual(sat.mo, 0.3373093125574321)
         self.assertEqual(sat.no_kozai, 0.04722944544077857)
 
+    def test_satrec_sgp4init_attributes(self):
+        # Make sure the Satrec has the same attributes if initialized via sgp4init() directly
+        satnum = 5
+        jdsatepoch = 2451723.28495062
+        bstar = 2.8098e-05
+        ndot = 6.96919666594958e-13
+        nddot = 0.0
+        ecco = 0.1859667
+        argpo = 5.790416027488515
+        inclo = 0.5980929187319208
+        mo = 0.3373093125574321
+        no_kozai = 0.04722944544077857
+        nodeo = 6.08638547138321
+
+        # Make sure the Satrec ininitalized via sgp4init() has the same attributes.
+        sat = self.build_satrec_sgp4init(satnum, jdsatepoch, bstar, ndot, nddot,
+                          ecco, argpo, inclo, mo, no_kozai, nodeo)
+        self.assertEqual(sat.satnum, 5)
+        # sgp4init does not back-calculate epochdays - does it need to?
+        #self.assertEqual(sat.epochdays, 179.78495062)
+
+        ## sgp4init does not set jdsatepoch internally  
```

Comment:
Feel free to remove this whole comment block — but thanks for leaving it here for me to make a decision about.

Reply:
Removed.